.github/workflows/ftl-build.yml

```diff
@@ -15,83 +18,176 @@ on:
     # 1:30am UTC every Sunday, has no particular significance
     - cron: "30 1 * * 0"
 
+env:
+  DOCKER_REGISTRY_IMAGE: ${{ secrets.DOCKERHUB_NAMESPACE }}/ftl-build
+  GITHUB_REGISTRY_IMAGE: ghcr.io/${{ github.repository_owner }}/ftl-build
+
 jobs:
-  build:
+  build-and-test:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        ARCH: [aarch64, armv4t, armv5te, armv6hf, armv7hf, armv8a, x86_32, x86_64, x86_64-musl, riscv64]
+        include:
+          - platform: linux/amd64
+            container: alpine:3.18
+          - platform: linux/386
+            container: alpine:3.18
+          - platform: linux/arm/v5
```

Comment:
Should the debian ones not be `linux/arm/v4` and `linux/arm/v5`? Seems we have `linux/arm/v6` twice...

Reply:
I just remember a very confusing conversation about this.
Pinging: @DL6ER

Reply:
Yes, because Debian always builds for one architecture *lower*. In the sense of "if you have an armv6 you can definitely run armv5 binaries". Alpine doesn't do this.

That's why

- `debian/armv5` actually builds `armv4` binaries,
- `debian/armv6` actually builds `armv5` binaries, however
- `alpine/armv6` actually builds `armv6`